nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,108 @@
+#ICA_AROMA pulled from: https://github.com/rhr-pruim/ICA-AROMA
+#This assumes ICA_AROMA.py is already installed and callable via $PATH
```

Comment:
@effigies, @jdkent - we are missing the standard header that imports the testing paths for doctest. see any other interface file in nipype.